bot/cogs/tags.py

```diff
@@ -86,11 +86,62 @@ def _get_suggestions(self, tag_name: str, thresholds: Optional[List[int]] = None
             return self._get_suggestions(tag_name)
         return found
 
+    async def _get_tags_via_content(self, check: Callable[[Iterable], bool], keywords: str) -> Optional[Embed]:
+        """
+        Search for tags via contents.
+
+        `predicate` will be either any or all, or a custom callable to search. Must return a bool.
+        """
+        await self._get_tags()
+
+        keywords_processed: Tuple[str] = tuple(query.strip().casefold() for query in keywords.split(',') if query)
+        keywords_processed = keywords_processed or (keywords,)
+        founds: list = [
+            tag
+            for tag in self._cache.values()
+            if check(query in tag['embed']['description'] for query in keywords_processed)
+        ]
+
+        if not founds:
```

Comment:
`if founds` feels semantically weird. Maybe this variable can be renamed so that this line sounds more semantically correct. I would've used something like `matching_tags` or `matches`.

Reply:
Since it was a simple `search and return` I decided to name it `founds` - which indeed hide away what it is unintentionally, when in reality it is a simple list of `tag`.
I really like your suggestion of `matching_tags`!